pkg/controller/suite_test.go

```diff
@@ -125,14 +143,102 @@ func (f *FakeMonitorManager) RegisterMonitor(workloadType string,
 func (f *FakeMonitorManager) DeregisterMonitor(workload types.NamespacedName) {}
 func (f *FakeMonitorManager) Shutdown()                                       {}
 
-type FakePolicyStore struct{}
+type FakePolicyStore struct {
```

Comment:
For the purpose of testing a mock impl of PolicyStore should be fine. We do not need to insert actual policies?

Reply:
I wanted the store interface methods to get tested as part of this instead of returning some hardcoded policy.